mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -219,6 +220,7 @@ def create_registered_model(self, name, tags=None, description=None):
         :return: A single object of :py:class:`mlflow.entities.model_registry.RegisteredModel`
                  created in the backend.
         """
+        _validate_model_name(name)
```

Comment:
we do not need a validation here because validation will be done in the backend server side (i.e., `SqlalchemyStore` and `FileStore`)